test/functional/books_controller_test.rb

```diff
@@ -21,7 +21,7 @@ class BooksControllerTest < ActionController::TestCase
   end
 
   test "test book strong params" do
-    post :create, { id: 1}
+    post :create, params: { call_number: "call_number" }
```

Comment:
Why did this change from using `id` to `call_number`?

(At some point this test could be improved by adding some more detailed assertions, but fine for now.)

Reply:
Just created book with another params instead of id.

Reply:
Oh okay — sounds good. Thanks!